# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: media/mtransport/nr_socket_tcp.cpp
# Commit: c43589fd0f90
# Full Hash: c43589fd0f90d174eb185b563406fd0a2e6ea712
# Author: Byron Campen [:bwc] <docfaraday@gmail.com>
# Date: 2019-09-18 21:49:27
# Regressor Bug: 1569183
# File Overlap Count: 2
# Description:
#   Bug 1569183: Update OnConnected callbacks to specify what type of proxying (if any) is being used, and implement IsProxied based on this information. r=mjf
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D45186
# ==============================================================================

diff -r 940d7f787bea -r c43589fd0f90 media/mtransport/nr_socket_tcp.cpp
--- a/media/mtransport/nr_socket_tcp.cpp	Tue Sep 17 18:18:15 2019 +0000
+++ b/media/mtransport/nr_socket_tcp.cpp	Tue Sep 17 18:18:33 2019 +0000
@@ -246,8 +246,9 @@
   DoCallbacks();
 }
 
-void NrTcpSocket::OnConnected() {
+void NrTcpSocket::OnConnected(const nsCString& aProxyType) {
   r_log(LOG_GENERIC, LOG_DEBUG, "NrTcpSocket::OnConnected %p\n", this);
+  mProxyType = aProxyType;
 
   DoCallbacks();
 }